catalog-info.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: vets-api
+  description: This project provides common APIs for applications that live on VA.gov (formerly vets.gov APIs).
+  annotations:
+    github.com/project-slug: department-of-veterans-affairs/vets-api
+    backstage.io/techdocs-ref: dir:.
+  tags:
+    - ruby
+    - rails
+spec:
+  type: website
+  owner: vsp-tools-backend
```

Comment:
GitHub team that owns this product.

Reply:
This team name may change in the coming days/week.